.github/workflows/auto_update.yml

```diff
@@ -0,0 +1,52 @@
+name: Update DB
+
+on: workflow_dispatch
+
+jobs:
+  update_db:
+    name: "Update DB"
+    runs-on: ubuntu-latest
+    env:
+      IMAGEBUILDER_BOT_GITLAB_SSH_KEY: ${{ secrets.IMAGEBUILDER_BOT_GITLAB_SSH_KEY }}
+      TRIGGER_TOKEN: ${{ secrets.TRIGGER_TOKEN }}
+    steps:
+      - name: Clone composer
+        uses: actions/checkout@v3
+        with:
+          repository: osbuild/osbuild-composer
+          path: composer
+
+      - name: Checkout manifest-db
+        uses: actions/checkout@v3
+        with:
+          path: manifest-db
+          fetch-depth: 0
+
+      - name: Update DB
+        run: |
+          ./manifest-db/tools/import-image-tests --verbose composer/test/data/manifests/ manifest-db/manifest-db --manifest-only --db-ignore=manifest-db/db-ignore
+
+      - name: Send to CI
+        run: |
+          cd manifest-db
+          git config --local user.name "SchutzBot"
+          git config --local user.email "schutzbot@redhat.com"
+
+          now=$(date '+%Y-%m-%d-%H%M%S')
+          BRANCH_NAME="db-update$now"
+          git branch "$BRANCH_NAME"
+          git checkout "$BRANCH_NAME"
+          git add -A
+          git commit -m "db: update manifests
```

Comment:
What about putting `now` into the commit message? Is that a bad idea?

Reply:
So, the idea is to have the branch name to be every time different. So we don't run into the issue of recycling branches on gitlab and github. As for integrating it in the commit message, I think we can just rely on the commit date.